### PR TITLE
SO-4383: Concept suggestion test cases

### PR DESCRIPTION
**Pull request by kovacstomi**

### CODE REVIEW

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java [hunk 1]
```diff
@@ -119,8 +122,10 @@
...
 					.setLocales(Strings.isNullOrEmpty(params.getAcceptLanguage()) ? acceptLanguage : params.getAcceptLanguage())
 					.setPreferredDisplay(params.getPreferredDisplay())
-					.filterByTerm(params.getTerm())
 					.setMinOccurrenceCount(params.getMinOccurrenceCount())
+					.filterByTerm(params.getTerm())
+					.filterByQuery(params.getQuery())
+					.filterByExclusion(params.getMustNotQuery())
 					.sortBy(SORT_BY)
 					.build(params.getCodeSystemPath())
 					.execute(getBus());
```
Comment (apeteri) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java]: Opportunity for refactoring: this block looks to be the same as the one in `postSuggest`. I'd either extract this part to a private method and call it from both places, or delegate to `postSuggest` directly from here.

**Review by apeteri — APPROVED** [review]
👍

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java (hunk 1 above)
Comment (kovacstomi) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java]: Simplified the code according to your suggestions. Thanks! 🎉

**Review by apeteri — APPROVED** [review]

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**